packages/tests/server/react/polymorphism.test.tsx

```diff
@@ -262,86 +245,6 @@ describe('polymorphism', () => {
       });
     });
   });
-
```

Comment:
It does not look like this kind of subtyping (the input/output being subtyped) can be possible in the new world, but that's probably okay because it's a much more complex style of polymorphism that probably warrants WET over DRY to reduce userland complexity